src/makeCalculator.js

```diff
@@ -4,7 +4,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(operation, num) {
+      this.result = operation(this.result, num);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => {
```

Comment:
The arithmetic functions (`add`, `subtract`, `multiply`, `divide`) should be methods that operate on the `result` property using `this.result`. This will help maintain the state within the calculator object and improve the overall design. Consider refactoring these functions to be methods of the calculator object.